installer/conf/td-agent-bit.conf

```diff
@@ -14,6 +14,15 @@
     Path_Key filepath
     Skip_Long_Lines On
 
+[INPUT]
+    Name tail
+    Tag oms.container.log.flbplugin.*
```

Comment:
the oms.container.log.* tails omsagent*.log as well. its that we need to send this to our telemetry AI (and also to customers workspace if kube-sys log collection is enabled). should we need anither tailer which is doing subset of the already exsting one? In this case, we can simply check the file name and route to AI (and also to workspace if kube-sys collection is enabled).

Reply:
As discussed offline, this way, we can keep the logs and telemetry separate by having two separate tailers